core/services/ocr2/validate/validate.go

```diff
@@ -176,3 +179,29 @@ func validateOCR2VRFSpec(jsonConfig job.JSONConfig) error {
 	}
 	return nil
 }
+
+func validateOCR2KeeperSpec(jsonConfig job.JSONConfig) error {
```

Comment:
This function should also be paired with a unit test. Right now the plugin config is just doing marshal/unmarshal logic which will make the test easy, but it's best to set the expectation early on what this function should do.